Tests/AppcuesKitTests/Actions/AppcuesContinueActionTests.swift

```diff
@@ -59,3 +59,16 @@ class AppcuesContinueActionTests: XCTestCase {
         XCTAssertEqual(showStepCount, 1)
     }
 }
+
+extension AppcuesContinueAction {
+    convenience init?() {
+        self.init(configuration: ExperiencePluginConfiguration(nil))
+    }
+    convenience init?(index: Int? = nil, offset: Int? = nil, stepID: String? = nil) {
+        var uuid: UUID?
+        if let stepID = stepID {
+            uuid = UUID(uuidString: stepID)
+        }
+        self.init(configuration: ExperiencePluginConfiguration(AppcuesContinueAction.Config(index: index, offset: offset, stepID: uuid)))
+    }
```

Comment:
made various convenience inits on Actions and Traits like this in tests to just encapsulate the usage of the underlying Config type and simplify testing callsites.

Reply:
I'm not sure I agree with doing this everywhere (specifically in the various `testInit()` tests. Reason being in these tests we're explicitly asserting that, for example, a `nil` init does _whatever_, but adding the layer of indirection with a testing-only convenience init makes it harder to understand what the test is actually testing. For these cases, I'd prefer to keep `AppcuesContinueAction(configuration: ExperiencePluginConfiguration(nil))` or whatever.

Absolutely makes sense for test-only actions/traits and any tests that aren't specifically testing the initializer.

Reply:
I'm not sure I agree with doing this everywhere (specifically in the various `testInit()` tests. Reason being in these tests we're explicitly asserting that, for example, a `nil` init does _whatever_, but adding the layer of indirection with a testing-only convenience init makes it harder to understand what the test is actually testing. For these cases, I'd prefer to keep `AppcuesContinueAction(configuration: ExperiencePluginConfiguration(nil))` or whatever.

Absolutely makes sense for test-only actions/traits and any tests that aren't specifically testing the initializer.